setup.py

```diff
@@ -217,6 +260,90 @@ def append_nvcc_threads(nvcc_extra_args):
             ],
         )
     )
+elif not SKIP_CUDA_BUILD and IS_ROCM:
+    ck_dir = "csrc/composable_kernel"
+
+    #use codegen get code dispatch
+    if not os.path.exists("./build"):
+        os.makedirs("build")
+
+    os.system(f"python3 {ck_dir}/example/ck_tile/01_fmha/generate.py --output_dir build --receipt 2")
+
+    print("\n\ntorch.__version__  = {}\n\n".format(torch.__version__))
+    TORCH_MAJOR = int(torch.__version__.split(".")[0])
+    TORCH_MINOR = int(torch.__version__.split(".")[1])
+
+    # Check, if ATen/CUDAGeneratorImpl.h is found, otherwise use ATen/cuda/CUDAGeneratorImpl.h
+    # See https://github.com/pytorch/pytorch/pull/70650
+    generator_flag = []
+    torch_dir = torch.__path__[0]
+    if os.path.exists(os.path.join(torch_dir, "include", "ATen", "CUDAGeneratorImpl.h")):
+        generator_flag = ["-DOLD_GENERATOR_PATH"]
+
+
+    check_if_rocm_home_none("flash_attn")
+    # Check, if CUDA11 is installed for compute capability 8.0
```

Comment:
forgot to delete?

Reply:
85692866ab88f006c6d7ce3c5009129007a70108